commands/operator-sdk/cmd/new.go

```diff
@@ -121,20 +121,25 @@ func mustBeNewProject() {
 }
 
 // repoPath checks if this project's repository path is rooted under $GOPATH and returns project's repository path.
-func repoPath() string {
-	gp := os.Getenv(gopath)
-	if len(gp) == 0 {
-		cmdError.ExitWithError(cmdError.ExitError, fmt.Errorf("$GOPATH env not set"))
-	}
+// repoPath field on generator is used primarily in generation of Go operator. For Ansible we will set it to cwd
+func repoPath(operatorType string) string {
```

Comment:
I do not believe you need to pass this into the function, you should be able to get it as it is a package var

Reply:
maybe we should have a enum for operatorType instead of string?